core/src/raw/tests/utils.rs

```diff
@@ -88,3 +91,69 @@ pub fn init_test_service() -> Result<Option<Operator>> {
 
     Ok(Some(op))
 }
+
+/// A logging interceptor that logs the backtrace.
+#[derive(Debug, Clone)]
+struct BacktraceLoggingInterceptor;
+
+impl LoggingInterceptor for BacktraceLoggingInterceptor {
+    fn log(
+        &self,
+        scheme: Scheme,
+        operation: raw::Operation,
+        path: &str,
```

Comment:
How about merging `path` into `context` as well? I believe this makes more sense, especially for cases where the path is empty or requests involve more than one path.

Reply:
> I believe this makes more sense

Yes. The only problem is that if we want to output `path={}` then we always need to allocate a string for it additionally. So I keep the path separated.

Reply:
I have two ideas:

- keep only one `context: &str`. We need to format a string anyway so adding an extra path inside should not have big difference.
- Change context into `&[(&str, &str)]` to avoid extra allocations.

Reply:
Let's keep things simple. I'll merge `path` into `context: &str`.